datadog_checks_dev/datadog_checks/dev/tooling/e2e/agent.py

```diff
@@ -26,7 +26,7 @@
 
 def get_rate_flag(agent_version):
     if agent_version >= 6:
-        return '--check-rate'
+        return '--rate'
```

Comment:
> but --check-rate is not a valid argument

The argument seems correct to me:

```
docker run -it datadog/agent:7.16.0-rc.4 agent check --help | grep check-rate
  -r, --check-rate          check rates by running the check twice with a 1sec-pause between the 2 runs
```

In which case you saw the argument was not valid ?

Reply:
Maybe confusion with the `ddev env check` argument ?

```
$ ddev env check --help | grep rate
  -r, --rate                Compute rates by running the check twice with a
```

Reply:
https://github.com/DataDog/datadog-agent/blob/1c117b0c6f256ffdf6512c18ccd8eac6e809b295/cmd/agent/app/check.go#L370

Reply:
Yes, I got this message running `ddev env check`... so maybe the method should be context aware :/

Reply:
There are 2:

https://github.com/DataDog/datadog-agent/blob/1c117b0c6f256ffdf6512c18ccd8eac6e809b295/cmd/agent/app/check.go#L370
https://github.com/DataDog/integrations-core/blob/b2ad560841758d77e24d4673e576b00621b7e637/datadog_checks_dev/datadog_checks/dev/tooling/commands/env/check.py#L78